src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -4400,17 +4400,17 @@
     </Field>
     <Field Name="IsActive" Type="bool" Override="true"/>
   </Node>
-  <Node Name="NonNullDirectiveTriviaSyntax" Base="DirectiveTriviaSyntax">
-    <Kind Name="NonNullDirectiveTrivia"/>
+  <Node Name="NullableDirectiveTriviaSyntax" Base="DirectiveTriviaSyntax">
+    <Kind Name="NullableDirectiveTrivia"/>
     <Field Name="HashToken" Type="SyntaxToken" Override="true">
       <Kind Name="HashToken"/>
     </Field>
-    <Field Name="NonNullKeyword" Type="SyntaxToken">
-      <Kind Name="NonNullKeyword"/>
+    <Field Name="NullableKeyword" Type="SyntaxToken">
+      <Kind Name="NullableKeyword"/>
     </Field>
-    <Field Name="DisableOrRestoreKeyword" Type="SyntaxToken">
+    <Field Name="SettingKeyword" Type="SyntaxToken">
```

Comment:
The naming signifies a `setting` keyword. That was not practiced before.

Reply:
Good point. How about `SettingToken`?

---
In reply to: [226397009](https://github.com/dotnet/roslyn/pull/30570#discussion_r226397009) [](ancestors = 226397009)

Reply:
This valid values for this token may be { `enable`, `disable`, `restore` } in the future. Will go with `SettingToken` to avoid `EnableOrDisableOrRestoreKeyword`.

---
In reply to: [226416106](https://github.com/dotnet/roslyn/pull/30570#discussion_r226416106) [](ancestors = 226416106)

Reply:
Right. Looks like `<Kind Name="RestoreKeyword"/>` was accidentally removed under this node.

Reply:
`restore` is not supported currently. What was previously called `restore` is now called `enable`.

---
In reply to: [226426914](https://github.com/dotnet/roslyn/pull/30570#discussion_r226426914) [](ancestors = 226426914)